js/load-image.js

```diff
@@ -159,10 +162,15 @@
 
   loadImage.onload = function (img, event, file, url, callback, options) {
     revokeHelper(url, options)
-    loadImage.transform(img, options, callback, file, {
-      originalWidth: img.naturalWidth || img.width,
-      originalHeight: img.naturalHeight || img.height
-    })
+
+    try {
+      loadImage.transform(img, options, callback, file, {
+        originalWidth: img.naturalWidth || img.width,
+        originalHeight: img.naturalHeight || img.height
+      })
+    } catch (error) {
+      callback(error)
+    }
```

Comment:
@blueimp This should now hanlde any uncaught exceptions in any transformations, if I understood the system correctly 🤔